shopify-client/src/App.vue

```diff
@@ -1,10 +1,9 @@
 <template>
   <div id="app">
     <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
```

Comment:
I think you need router-link and router-view for the router to work. https://router.vuejs.org/api/#router-link

Reply:
`<b-nav-item>` is sufficient for navigating to different views.